### ISSUE TITLE
Double title bars

### ISSUE THREAD

**Issue by kattjevfel** [0x90d/videoduplicatefinder#375]
**Environment**
 - OS: Arch Linux

**Describe the bug**
As of 032a5c0ee1abc81b1daeda7d383ac3f3bf0302af at least on KDE Plasma you get two title bars, as opposed to just the system native one like before.
![2023-01-21_22-09-36](https://user-images.githubusercontent.com/6226312/213887388-7da44f72-aab8-425a-85d6-3e678cc42d83.png)

**Expected behavior**
Only one title bar should be visible.

**Comment by 0x90d** [comment]
Are you sure that its because of THIS commit? All it does is horizontally aligning the title text. But your screenshot shows that your Linux doesn't support the chrome windows. Anyway this is definitely an Avalonia issue (or not an issue at all if your Linux doesn't support it).

**Comment by kattjevfel** [comment]
No, it is most likely not that exact commit, just the one I tested. Either way I must apologise as I realised there was always a double title bar, just that it took up less space and I wasn't thinking about it.
![2023-01-21_22-27-37](https://user-images.githubusercontent.com/6226312/213887776-e5d84a66-5dc8-44c8-a0c8-8ab2e2e2edb9.png)

tl;dr Tired and confused, I was mistaken.
